app/views/scheduled_pickups/show.html.erb

```diff
@@ -5,7 +5,7 @@
   </header>
   <section class="zipcode-content-section">
     <h2 class="section-label">Scheduled Pickup Time</h2>
-    <span><%= format_pickup_time(@scheduled_pickup) %></span>
+    <span><%= @scheduled_pickup.time_range %></span>
```

Comment:
Is there a feature spec that checks that this is being displayed how you expect?